src/test/analyzer_test.ts

```diff
@@ -232,16 +233,63 @@ suite('Analyzer', () => {
     });
 
     const analyzer = new BuildAnalyzer(config);
+    assert.isFalse(analyzer.started);
+    analyzer.sources().pipe(new NoopStream());
+    assert.isTrue(analyzer.started);
+  });
+
+
+  test('calling dependencies() starts analysis', () => {
```

Comment:
These three tests (two new and 1 updated) are the only testing changes made by me. The rest of this `test/` diff is due to clang-formatting after the `.sources -> .sources()` & `dependencies -> dependencies()` change.